### PR TITLE
Update ESmy019.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
MY-019 reworked

### CODE REVIEW

File: ES/ESmy019.xml [hunk 1]
```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                 <incipit xml:lang="gez"/>
+                                 <explicit xml:lang="gez"></explicit>
+                                 <note>
+                                   For the edition, s. <bibl><ptr target="bm:Bausi2003Libanos"/>
+                                   <citedRange unit="paragraph">282-283</citedRange></bibl>, and
+                                   <bibl><ptr target="bm:Bausi2003Libanos"/>
+                                   <citedRange unit="paragraph">290-291</citedRange></bibl>.
+                                   The red ink of  is faded and poorly readable.
+                                   The subscription (<locus from="18vb" to="19ra"/>) states
+                                   that the manuscript was purchased and donated to the church of Dabra
```
Comment (eu-genia) [inline ES/ESmy019.xml:143]: you need the subscription as an extra element
`<explicit type="subscriptio"><locus from="18vb" to="19ra"/><note>The subscription states...</note></explicit>`

File: ES/ESmy019.xml [hunk 2]
```diff
@@ -99,117 +189,180 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+
+                  <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#1va">f. 1va</locus>
-                           <desc>-vb: A miracle of Gabra Manfas Qǝddus.</desc>
-                           <quote xml:lang="gez"/>
-                           <quote xml:lang="en"/>
-                           <note> Written in a crude hand. </note>
+                           <locus target="#1va"/>
```
Comment (eu-genia) [inline ES/ESmy019.xml:229]: replace target="#1va" with from="1va" to"1vb" as in the original description

File: ES/ESmy019.xml [hunk 3]
```diff
@@ -99,117 +189,180 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <desc type="GuestText"><title ref="LIT3977Taammer">One miracle of Gabra Manfas Qǝddus.</title></desc>
+                           <quote xml:lang="gez">ተአምሪሁ፡ ለአቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡ ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡
```
Comment (eu-genia) [inline ES/ESmy019.xml:231]: replace `<quote>` (not accepted by the schema) with `<q>`

File: ES/ESmy019.xml [hunk 4]
```diff
@@ -99,117 +189,180 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                             ተስፋ፡ ማርያም፡ ለዓለ፡ ወነበረ፡ ፩ነዳይ፡ ዘአልቦቱ፡ ጥሪት፡ ዘእንበለ፡ አሐዱ፡ ዶርሖ፡</quote>
+                           <quote xml:lang="en"></quote>
```
Comment (eu-genia) [inline ES/ESmy019.xml:233]: remove line 233

File: ES/ESmy019.xml [hunk 5]
```diff
@@ -99,117 +189,180 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <note>For the edition, s. <bibl><ptr target="bm:Marrassini2003GabraManfasQeddusTextus"/></bibl>
```
Comment (eu-genia) [inline ES/ESmy019.xml:234]: bibliography is for work, not for ms

File: ES/ESmy019.xml [hunk 6]
```diff
@@ -99,117 +189,180 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           and <bibl><ptr target="bm:Marrassini2003GabraManfasQeddusVersio"/>
+                           <citedRange unit="chapter">“Miracolo xii”</citedRange></bibl>.
+                           Starting from <ref target="#1vb"/>, l.5 the scribe (<ref target="#h1">Hand 1</ref>) used a different,
+                           thinner pen and wrote in smaller script.
+                         </note>
                         </item>
-                        <item xml:id="a2">
```
Comment (eu-genia) [inline ES/ESmy019.xml:149]: I believe this addition got lost?
(NB locus from="18vb" to "19ra")

File: ES/ESmy019.xml [hunk 7]
```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                   <citedRange unit="paragraph">290-291</citedRange></bibl>.
+                                   The red ink of  is faded and poorly readable.
+                                   The subscription (<locus from="18vb" to="19ra"/>) states
+                                   that the manuscript was purchased and donated to the church of Dabra
+                                   Maʿṣo by <persName ref="PRS13128GadlaGiyorgis">Gadla Giyorgis</persName> and
+                                   <persName role="donor" ref="PRS13127TasfaMaryam">Tasfā Māryām</persName>. “Condemnatio” mentions
+                                   <persName ref="PRS8388SalamaI">ʾabuna Salāmā III</persName> (in tenure <date>1841-67</date>).
+                                    </note>
+                                 </msItem>
+                                 <msItem xml:id="p1_i3">
```
Comment (eu-genia) [inline ES/ESmy019.xml:149]: is it sure a msItem and not guest text? it comes after a donation note, looks like an addition to me

File: ES/ESmy019.xml [hunk 8]
```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <collection>Ethio-SPaRe</collection>
                   <idno>MY-019</idno>
                </msIdentifier>
-               <msContents>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:34]: Now the summary does not correspond to the description. Is there any need in it? Each time someone will have to harmonize it with the description below.

File: ES/ESmy019.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:149]: Before it was an addition, indeed, but it is finely written and it it is an independent  record now. It must not be always an addition, it is rather a boundary case. We highlight a scribal hand which is not that of a student (now I do not remember if it attested elsewhere in MY mss). BUt if you insist, i can scale it down to the  addition

File: ES/ESmy019.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:229]: done

File: ES/ESmy019.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:231]: done

File: ES/ESmy019.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:234]: ok

File: ES/ESmy019.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:233]: sorry, do not understand, what was in line 233?

File: ES/ESmy019.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:149]: Before, they both were one "additio". In fact, I do not know what it is. Magic? Writing exercise?

File: ES/ESmy019.xml (hunk 8 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESmy019.xml:34]: you still need the empty msContents+summary elements in order to have the automatically generated summary of the MS

File: ES/ESmy019.xml (hunk 4 above)
Comment (eu-genia) [reply ES/ESmy019.xml]: `                           <quote xml:lang="en"></quote>
`

File: ES/ESmy019.xml (hunk 8 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:34]: ok, done

File: ES/ESmy019.xml [hunk 9]
```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <msContents>
+                           <summary/>
+
+                           <msItem xml:id="p1_i1">
+                              <locus from="2ra" to="17va"/>
+                              <title type="complete" ref="LIT6166MiraclesJohnBaptist">Ten
+                                miracles of John the Baptist</title>
+                              <incipit xml:lang="gez"/>
+                              <note>
+                                  Cp. Mss Gunda Gunde GG-024, RQM-029.
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml]: I will move this to the work.

File: ES/ESmy019.xml [hunk 10]
```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                              </note>
+                              </msItem>
+
+                              <msItem xml:id="p1_i2">
+                                 <locus from="17vb" to="19rb"></locus>
+                                 <title type="complete" ref="LIT4717TaammerLi">Two miracles of Libānos</title>
+                                 <incipit xml:lang="gez"/>
+                                 <explicit xml:lang="gez"></explicit>
+                                 <note>
+                                   For the edition, s. <bibl><ptr target="bm:Bausi2003Libanos"/>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml]: I'll do this.

File: ES/ESmy019.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:136]: Not sure I understood it right.

File: ES/ESmy021.xml [hunk 11]
```diff
@@ -216,20 +314,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
          </particDesc>
          <textClass>
             <keywords scheme="#ethioauthlist">
                <term key="Missal"/>
+               <term key="ZaMa"/>
+               <term key="MoPe"/>
+               <term key="Liturgy"/>
             </keywords>
          </textClass>
          <langUsage><language ident="en">English</language></langUsage>
```
Comment (eu-genia) [inline ES/ESmy021.xml:329]: complete langUsage

File: ES/ESmy021.xml [hunk 12]
```diff
@@ -247,4 +341,4 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <ab/>
       </body>
    </text>
-</TEI>
```
Comment (eu-genia) [inline ES/ESmy021.xml:250]: add `<change>` element (in both files - and all the others)

File: ES/ESmy021.xml [hunk 13]
```diff
@@ -103,82 +175,108 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                     <decoNote type="miniature" xml:id="d2"/>
+                     <decoNote type="band" xml:id="d1">
+                      <locus target="#19r"/>
+                      <desc>Two crude narrow coloured (red, green, black) ornamental bands
+                        over the text columns, at the incipit of <ref target="#ms_i1.5"/>.</desc>
+                      </decoNote>
                   </decoDesc>
                   <additions>
                      <list>
-                        <item xml:id="e1">- Text is written in (smaller letters and) compressed script: fols. 15r-v, 17r. - Crude musical notation signs on fols. 19r, 22r-23v and elsewhere. - Crudely drawn lines delimiting sections of the text: e.g., fols. 20rb, 29ra, 30ra.</item>
```
Comment (eu-genia) [inline ES/ESmy021.xml:153]: is the content of other extras lost or moved somewhere?

File: ES/ESmy019.xml [hunk 14]
```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                                 <locus from="12r" to="17v"/> II(fols. 12r-17v) </item>
-                              <item xml:id="q4">
-                                 <locus from="18r" to="20v"/> III(fols. 18r-20v) </item>
-                              <item xml:id="q5">
-                                 <locus from="21r" to="30v"/> &lt;IV(fols. 21r-30v) </item>
-                              <item xml:id="q6">
-                                 <locus from="31r" to="40v"/> V(fols. 31r-40v) </item>
+                              <item xml:id="q1" n="A">
+                                <dim unit="leaf">1</dim>
+                                 <locus target="#1r"/>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml:59]: done

File: ES/ESmy021.xml (hunk 11 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy021.xml:329]: done

File: ES/ESmy021.xml (hunk 12 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy021.xml:250]: ok

File: ES/ESmy021.xml (hunk 13 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy021.xml:153]: removed

File: ES/ESmy019.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy019.xml]: ok

**Comment by DenisNosnitsin1970** [comment]
I will deal with Miracles of Gabra Manfas Qeddus, Libanos and Jesus later.

**Comment by DenisNosnitsin1970** [comment]
Sorry. Thanks!

**Pull request merged**